### ISSUE TITLE
Doesn't show up at all

### ISSUE THREAD

**Issue by aniforprez** [jakesankey/line-diff-details#13]
Clicking the line number does nothing. Is it because of conflicts with other packages?

**Comment by veegee2** [comment]
Pls take a look of it. I have also the same problem.

**Comment by jakesankey** [comment]
It may be a conflict with another package.. It isn't something I am able to reproduce without more context.. are you using a beta version of atom or the regular release? What version?

**Comment by veegee2** [comment]
Now i have version 1.3.3. Installed packages from config: "atom-beautify", "atom-minify",  "one-dark-ui",  "autocomplete-plus",  "find-and-replace", "spell-check",  pigments,   "one-light-ui",  "project-manager",  "highlight-selected",  "merge-conflicts",  "git-history",   "tool-bar",  "git-diff",  "fuzzy-finder",  "markdown-preview",  "color-picker",  "terminal-panel",  "atom-terminal-panel",   "atom-material-ui".
DisabledPackages: ["language-c",  "language-clojure", "language-csharp", "language-gfm",    "language-objective-c", "language-mustache", "language-perl", "language-python", language-ruby",    "language-ruby-on-rails", "language-shellscript",  "language-hyperlink",  "language-toml", "language-todo", "language-yaml", "metrics", "wrap-guide", "welcome", "grammar-selector", "language-java",     "language-go", "language-property-list", "link", "language-make", "markdown-preview", "open-on-github", "atom-jade", "react", ]

**Comment by veegee2** [comment]
After a new version of line-diff problem stay the same like before, say me please what do you need to know. (can i see some error code or something to identify the conflict?)

**Comment by teaman** [comment]
It does not work for me either. I'm using v1.5.3 of Atom.

**Comment by jakesankey** [comment]
Sorry folks, I have not been able to reproduce this. Please give me some additional context about your setup. Are you working with multiple git-based projects in a single atom window?

**Comment by Leistungsabfall** [comment]
I had the same problem. It worked for me as soon as I saved the file (Ctrl+S) and clicked the line number again.

**Comment by aniforprez** [comment]
Hey, I'm really sorry but I've switched out from Atom some time ago due to performance reasons so I won't be able to test this issue. It didn't work for any number of git projects from 1 to 5 (for a certain project I was doing). My feeling was some other line package was conflicting but I've uninstalled atom since then and I'm not going to install it all again just to reproduce this issue. Sorry again that I can't help.
